keyboards/split75/README.md

```diff
@@ -0,0 +1,48 @@
+split75 ps2avrGB keyboard firmware
```

Comment:
Personally I would like to see this readme aligned with the [template file](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/ps2avrgb/readme.md).

Reply:
I've updated it but not sure what to put in the `Hardware Supported` field, sorry 😞

Reply:
If this port were to work on multiple boards, that could be worth noting ( like this would work on the official and clones?). Otherwise what you have right now is fine with me.

Reply:
I will have to assume that this won't work on the other boards since this board is split. Either way can't test it as I don't have access to other keyboards, sorry!

Reply:
> since this board is split.

I was assuming similar hardware here.... I wouldnt expect this to work on a non split board. For example <https://github.com/qmk/qmk_firmware/tree/master/keyboards/melody96>

> The PCB for this board is also commonly used in the NYM96